sycl/test/Unit/lit.cfg.py

```diff
@@ -72,6 +72,5 @@ def find_shlibpath_var():
 else:
     lit_config.warning("unable to inject shared library path on '{}'"
                        .format(platform.system()))
-
-config.environment['SYCL_DEVICE_FILTER'] = lit_config.params.get('SYCL_PLUGIN', "opencl")
+config.environment['SYCL_DEVICE_FILTER'] = lit_config.params.get('SYCL_PLUGIN', "opencl,host")
```

Comment:
```suggestion
config.environment['SYCL_DEVICE_FILTER'] = lit_config.params.get('SYCL_PLUGIN', "opencl") + ",host"
```
SYCL_PLUGIN can be configured in CMakeLists.txt (e.g.  for cuda) and we need to append ",host" to its value to make it work as expected.